### ISSUE TITLE
Proxy Path

### ISSUE THREAD

**Issue by AndreGuerra123** [devforth/spa-to-http#12]
Hi, 

Currently, I need to add a proxy pass (designation under nginx) to /api, common in most vuejs projects. My case is to a external URL. Is this possible and how to do it?

**Comment by maybeanerd** [comment]
AFAIK this replaces nginx to server static assets, not as a reverse proxy. If you want to use another reverse proxy than nginx, try out https://traefik.io/traefik/

In general you would have a setup similar to
traefik
  backend (some image)
  frontend (served by this image)

and have traefik control what paths point to where

**Comment by ivictbor** [comment]
Yes, exactly, spa-to-http is generally designed for use with Traefik or when you . Traefik is simpler and easier to manage in most use-cases then Nginx. It has a lot of nice features like very simple and free HTTPS(SSL) setup ([Example](https://devforth.io/blog/onlogs-open-source-simplified-web-logs-viewer-for-dockers/)).

So in general a lot of developers used Traefik for proxying API/complex schemes, but served SPA bundles with things like Nginx (in combination with Traefik). So we creates SPA-to-HTTP as dedicated and faster alternative to Nginx, which again could be used in combination with Traefik. You can read [detailed Traefik vs Nginx comparison guide](https://devforth.io/blog/nginx-vs-traefik-how-slower-one-can-be-better/), there we explained why we switched to Traefik
